packages/terra-core-docs/src/terra-dev-site/doc/status-view/example/Custom.jsx

```diff
@@ -1,24 +1,38 @@
 import React from 'react';
-import IconDiamond from 'terra-icon/lib/icon/IconDiamond';
+import IconDueSoon from 'terra-icon/lib/icon/IconDueSoon';
 import StatusView from 'terra-status-view';
+import VisuallyHiddenText from 'terra-visually-hidden-text';
 
 const StatusViewButtons = [
   {
-    text: 'Action 1',
+    'aria-describedby': 'submittimesheet',
+    id: 'Submit',
     key: 1,
+    text: 'Submit',
+    variant: 'emphasis',
   }, {
-    text: 'Action 2',
+    'aria-describedby': 'canceltimesheet',
+    id: 'Cancel',
     key: 2,
+    text: 'Cancel',
   },
 ];
 
 const Custom = () => (
-  <StatusView
-    message="Custom Message"
-    customGlyph={<IconDiamond />}
-    title="Custom Title"
-    buttonAttrs={StatusViewButtons}
-  />
+  <div>
+    <VisuallyHiddenText aria-hidden id="submittimesheet" text="Select to Submit Timesheet" />
+    <VisuallyHiddenText aria-hidden id="canceltimesheet" text="Select to Cancel Timesheet" />
+    <StatusView
+      message="Once submitted cannot be revised"
+      customGlyph={<IconDueSoon />}
+      title="Timesheet"
+      buttonAttrs={StatusViewButtons}
+    />
+    <p>
+      Note: We should provide guidance for the action button labels. Those labels should give proper message or title in some way.
+      For Accessibility best practice one can use the ARIA attributes like aria-label to provide the context.
```

Comment:
This should be updated to use more consumer-centric language. Is "we" in this context Terra or the consumer?

Reply:
here it is terra consumers.

Reply:
Changes are made. Thanks